.gitignore

```diff
@@ -89,4 +89,5 @@ zookeeper-client/zookeeper-client-c/install-sh
 zookeeper-client/zookeeper-client-c/ltmain.sh
 zookeeper-client/zookeeper-client-c/missing
 zookeeper-server/src/main/java/org/apache/zookeeper/version/Info.java
+zookeeper-server/src/main/java/org/apache/zookeeper/version/VersionInfoMain.java
```

Comment:
I just noticed this issue during a build, good to fix. I agree with Norbert - this should be part of "generate-sources" phase/directory.